extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -48,9 +48,13 @@ amp-embed iframe {
   padding: 2px 4px;
 }
 
-amp-ad[type=adsense] > iframe {
+amp-ad[data-a4a-upgrade-type] > iframe {
+  width: 100%;
+  height: 100%;
   top: 50% !important;
   left: 50% !important;
   transform: translate(-50%, -50%);
+  min-width: 0;
+  min-height: 0;
```

Comment:
I'm not convinced this will work as the current problem is that the frame's height/width are 100%.  We actually want them to equal the size of the height/width attributes so I see no way you can do all of this from within the css file.  Will likely need to do from the impls themselves.